contracts/core/EntryPointSimulations.sol

```diff
@@ -172,10 +172,11 @@ contract EntryPointSimulations is EntryPoint, IEntryPointSimulations {
     // empiric test showed that without this wrapper, simulation depositTo costs less..
     function depositTo(address account) public override(IStakeManager, StakeManager) payable {
         unchecked{
-            uint g = gasleft();
-            uint x = block.number;
-            while (g - gasleft() < 120) {
-                x = x + block.number;
+        // silly code, to waste some gas to make sure depositTo is always little more
```

Comment:
Did you intend to rewrite this code again?

Reply:
yes, its simpler..